imap_processing/tests/hit/test_hit_l2.py

```diff
@@ -289,27 +298,63 @@ def test_calculate_intensities_for_a_species():
 
 
 def test_calculate_intensities():
-    """Test the calculate_intensities function."""
+    """Test the calculate_intensities function.
+
+    This tests the function with different values of seconds.
+    60 seconds is used for calculating standard and summed
+    intensities and 600 is used for sectored intensities.
+    """
     # Sample input data
     rate = xr.DataArray([100, 200, 300], dims=["energy_bin"])
     delta_e_factor = np.array([1.0, 1.0, 1.0])
     geometry_factor = np.array([1.0, 1.0, 1.0])
     efficiency = np.array([1.0, 1.0, 1.0])
     b = np.array([0.0, 0.0, 0.0])
+    seconds = 60
 
-    # Expected output
-    expected_intensities = xr.DataArray(
+    # Expected output for seconds = 60
+    expected_intensities_60 = xr.DataArray(
         [1.66666667, 3.33333333, 5.0], dims=["energy_bin"]
     )
 
-    # Call the function
-    intensities = calculate_intensities(
-        rate, delta_e_factor, geometry_factor, efficiency, b
+    factors_60 = IntensityFactors(
+        delta_e=delta_e_factor,
+        geometry_factor=geometry_factor,
+        efficiency=efficiency,
+        b=b,
+        seconds=seconds,
     )
 
-    # Assertions
-    assert np.allclose(intensities.values, expected_intensities.values), (
-        "Intensities mismatch"
+    # Call the function for seconds = 60
+    intensities_60 = calculate_intensities(rate, factors_60)
+
+    # Assertions for seconds = 60
+    assert np.allclose(intensities_60.values, expected_intensities_60.values), (
+        "Intensities mismatch for seconds = 60"
+    )
```

Comment:
Use the numpy testing assertions since they add helpful output about which quantities did not compare equal. Here and below.
```suggestion
    np.testing.assert_allclose(intensities_60, expected_intensities_60)
```